tunedev/Banka
# #164725527  create new account page and transaction full details page.

#### What does this PR do?
Adds two new pages to the ui folder

#### Description of Task to be completed?
Adds page create new account and view full transaction details to the user interface.

#### How should this be manually tested?
After cloning this repo, locate the create-account.html and transaction-details.html files, open them on any browser, it should be properly styled.

#### What are the relevant pivotal tracker stories?
[#164725527](https://www.pivotaltracker.com/story/show/164725527)
